tests/manager/manager-ctor-ssl-002.phpt

```diff
@@ -0,0 +1,17 @@
+--TEST--
+PHPC-1239: Passing SSL driverOptions overrides SSL options from URI
+--SKIPIF--
+<?php require __DIR__ . "/../utils/basic-skipif.inc"; ?>
+<?php skip_if_not_live(); ?>
+--FILE--
+<?php
+require_once __DIR__ . "/../utils/basic.inc";
+
+$manager = new MongoDB\Driver\Manager(URI, array(), array());
```

Comment:
We can use `[]` instead of `array()` here. Down the line, I'd like to go back and clean up all `array()` usages.

Reply:
We can fix that then then. It's ``array()`` for now here, just as in the original test.